### PR TITLE
Solution to Algorithm Challenge

### PR DESCRIPTION
**Pull request by Lenever**
All the test will pass.
Please review @bondz .

*Thanks in anticipation*

### CODE REVIEW

File: src/brackets/index.js [hunk 1]
```diff
@@ -4,6 +4,74 @@
...
+  }
+
+  if (
+    openBrackets.length != closedBrackets.length ||
+    str.includes("(]") ||
+    str.includes("(}") ||
+    str.includes("[)") ||
+    str.includes("[}") ||
+    str.includes("{)") ||
+    str.includes("{]") ||
```
Comment (bondz) [inline src/brackets/index.js:68]: Why are these specific cases included?

Comment (Lenever) [reply src/brackets/index.js:68]: I added them as a quicker way to filter through invalid bracket strings as specified in the README file. I tried removing them and the program fails the fifth test. Inclusion of these specific cases seems to be the only way to deal with cases of these kinds.

Comment (bondz) [reply src/brackets/index.js:68]: It's not. It's hardcoding for specific cases to pass the test and doesn't cover all scenarious. This isn't encouraged. I'll merge the PR though.

Comment (Lenever) [reply src/brackets/index.js:68]: I know it looks that way, but I assure you that was not my intention. Now that you've mentioned it, I'll work hard at other solutions that will eliminate appearances like this. Thanks.

**Pull request merged**